opmon-protobuf-dbwriter/dbwriter.py

```diff
@@ -0,0 +1,168 @@
+# @file dbwriter.py Writing Opmon entries into InfluxDB
+#  This is part of the DUNE DAQ software, copyright 2020.
+#  Licensing/copyright details are in the COPYING file that you should have
+#  received with this code.
+#
+
+import kafkaopmon.OpMonSubscriber as opmon_sub
+import google.protobuf.json_format as pb_json
+from google.protobuf.timestamp_pb2 import Timestamp
+import opmonlib.opmon_entry_pb2 as opmon_schema
+
+from influxdb import InfluxDBClient
+from functools import partial
+import json
+import click
+import logging
+import queue
+import threading
+
+
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+@click.command(context_settings=CONTEXT_SETTINGS)
+# subscriber options
+@click.option('--subscriber-bootstrap', type=click.STRING,                default="monkafka.cern.ch:30092", help="boostrap server and port of the OpMonSubscriber")
+@click.option('--subscriber-group',     type=click.STRING,                default=None, help='group ID of the OpMonSubscriber')
+@click.option('--subscriber-timeout',   type=click.INT,                   default=500, help='timeout in ms used in the OpMonSubscriber')
+@click.option('--subscriber-topic',     type=click.STRING, multiple=True, default=['opmon_stream'], help='The system will add the "monitoring." prefix' )
+
+#influx options
+@click.option('--influxdb-address', type=click.STRING, default='np04-srv-017', help='address of the influx db')
+@click.option('--influxdb-port',    type=click.INT,    default=31002,          help='port of the influxdb')
+@click.option('--influxdb-name',    type=click.STRING, default='test_influx',  help='Table name destination inside influxdb')
```

Comment:
Do we need a username/password also to permit write access?

Reply:
Is it a question for the microservice or for influx itself? At the moment this code should work. But if we need to add name and password we can probably add it.

Reply:
I don't think we've got influxdb configured for unauthenticated writes.

Reply:
Exactly, that's why this code will work as it is. Do we think it's a problem and do we want to fix it now? Or can we leave it for after the whole HD run?

Reply:
I'd be a bit surprised if the service was able to write to influxdb without the user/pass set.  I think args for those will need to be added.

Reply:
The old one did. But I agree that we can add them so the image is done once and for all. I'll do it first thing tomorrow.